doc/rados/operations/add-or-rm-mons.rst

```diff
@@ -126,6 +121,14 @@ on ``mon.a``).
 
 	ceph-mon -i {mon-id} --public-addr {ip:port}
 
+#. Add the new monitor to the list of monitors for you cluster (runtime). This enables
```

Comment:
s/you/your   (your cluster)

sorry to nitpick.  That was an old/existing typo.  Might as well fix it now!  :)